examples/Retrieval models (MovieLens 1M) - Youtube-DNN retrieval model example.ipynb

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
<p>I would not have words here like <code>sophisticated</code> and <code>novel architectures</code> as it is a classic architecture but a bit outdated now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>